airflow/www/static/js/dag.js

```diff
@@ -269,7 +271,7 @@ export function callModal({
   }
   query.delete('try_number');
 
-  if (extraLinks && extraLinks.length > 0) {
+  if (!isMapped && extraLinks && extraLinks.length > 0) {
```

Comment:
Should we also wrap this in a `!isMapped`?https://github.com/apache/airflow/blob/main/airflow/www/static/js/dag/details/taskInstance/index.tsx#L167

Reply:
I guess, I only fixed the view mentioned in #25360 because that’s the part I can understand :p